__tests__/server.test.js

```diff
@@ -266,4 +258,77 @@ describe("backend API project", () => {
         });
     });
   });
+  describe("PATCH /api/articles/:article_id", () => {
+    test("that a PATCH request responds with the updated article when given a correct body adding 1", () => {
+      return request(app)
+        .patch("/api/articles/1")
+        .send({
+          inc_votes: 1,
+        })
+        .expect(201)
+        .then(({ body }) => {
+          expect(typeof body.article).toBe("object");
+          expect(body.article).toHaveProperty("author", expect.any(String));
+          expect(body.article).toHaveProperty("title", expect.any(String));
+          expect(body.article).toHaveProperty("article_id", expect.any(Number));
+          expect(body.article).toHaveProperty("body", expect.any(String));
+          expect(body.article).toHaveProperty("topic", expect.any(String));
+          expect(body.article).toHaveProperty("created_at", expect.any(String));
+          expect(body.article).toHaveProperty("votes", 101);
+          expect(body.article).toHaveProperty(
+            "article_img_url",
+            expect.any(String)
+          );
+        });
+    });
+    test("that a PATCH request responds with the updated article when given a correct body subtracting 100", () => {
+      return request(app)
+        .patch("/api/articles/1")
+        .send({
+          inc_votes: -100,
+        })
+        .expect(201)
+        .then(({ body }) => {
+          expect(typeof body.article).toBe("object");
+          expect(body.article).toHaveProperty("author", expect.any(String));
+          expect(body.article).toHaveProperty("title", expect.any(String));
+          expect(body.article).toHaveProperty("article_id", expect.any(Number));
+          expect(body.article).toHaveProperty("body", expect.any(String));
+          expect(body.article).toHaveProperty("topic", expect.any(String));
+          expect(body.article).toHaveProperty("created_at", expect.any(String));
+          expect(body.article).toHaveProperty("votes", 0);
+          expect(body.article).toHaveProperty(
+            "article_img_url",
+            expect.any(String)
+          );
+        });
+    });
+    test("that a PATCH request with an invalid body type returns 400 Bad request", () => {
+      return request(app)
+        .patch("/api/articles/badrequest")
```

Comment:
This is not testing an invalid body type as you are testing an invalid path here instead.